src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Operations/Reindex/ReindexUtilitiesTests.cs

```diff
@@ -60,7 +67,7 @@ public async Task GivenResourcesWithUnchangedOrChangedIndices_WhenResultsProcess
             var resultList = new List<SearchResultEntry>();
             resultList.Add(entry1);
             resultList.Add(entry2);
-            var result = new SearchResult(resultList, "token", null, new List<Tuple<string, string>>());
+            var result = new SearchResult(resultList, "token", new List<(SearchParameterInfo, SortOrder)>(), new List<Tuple<string, string>>());
```

Comment:
>new List<(SearchParameterInfo, SortOrder)>() [](start = 63, length = 44)

Just curious, is this change because it unblocks something, or it's just for how nice code look this way?

Reply:
I think it was just for nice looking code :).